data/json/monstergroups/wilderness.json

```
@@ -1712,7 +1713,11 @@
       { "monster": "mon_aphid", "freq": 15, "pack_size": [ 1, 5 ], "cost_multiplier": 0 },
       { "monster": "mon_grasshopper_small", "freq": 15, "pack_size": [ 1, 3 ], "cost_multiplier": 0 },
       { "monster": "mon_antlion_larva", "freq": 7, "cost_multiplier": 10 },
-      { "monster": "mon_antlion_giant", "freq": 2, "cost_multiplier": 10 }
+      { "monster": "mon_antlion_giant", "freq": 2, "cost_multiplier": 10 },
+      { "monster": "mon_stag_beetle_small", "freq": 1, "cost_multiplier": 0, "ends": 504 },
+      { "monster": "mon_stag_beetle_small", "freq": 2, "cost_multiplier": 0, "starts": 168 },
+      { "monster": "mon_stag_beetle_giant", "freq": 2, "cost_multiplier": 0, "starts": 504 },
+      { "monster": "mon_stag_beetle_giant", "freq": 1, "cost_multiplier": 0, "starts": 720, "pack_size": [ 1, 2 ] }
```

```suggestion
      { "monster": "mon_stag_beetle_small", "freq": 2, "cost_multiplier": 0, "starts": 168, "ends": 720 },
      { "monster": "mon_stag_beetle_small", "freq": 2, "cost_multiplier": 0, "starts": 168 },
      { "monster": "mon_stag_beetle_giant", "freq": 1, "cost_multiplier": 0, "starts": 720 }
```
Natural-grown giants appear three months in, spawned ones three. The frequencies should be pretty low, players tend to generate more fields/forests than other terrain so they'll add up. They might need some more tuning, since they are pretty much untouchable by early-mid zombies and you shouldn't be able to cheese a whole town with a lucky spawn.